src/routes/Books.js

```diff
@@ -1,15 +1,20 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from '../components/Book';
 import Form from '../components/Form';
 
-const Books = () => (
-  <>
-    <ul>
-      <Book title="Rich Dad Poor Dad" author="Guy Kawasaki" />
-      <Book title="Shibumi" author="Christoph Grange" />
-      <Book title="Code of DaVinci" author="Dan Brown" />
-    </ul>
-    <Form />
-  </>
-);
+const Books = () => {
+  const booksArray = useSelector((state) => state.books);
+  return (
+    <>
+      {booksArray.map((item) => (
+        <div key={item.item_id}>
+          <Book id={item.item_id} title={item.title} author={item.author} />
+        </div>
+      ))}
+      <Form />
+    </>
```

Comment:
- [ ] The Form component called here is not showing on the screen for a book to be added to the store as expected. Kindly work on showing the form for books to be added so one is able to know if a book can be added to the store 👍🏽

<img width="636" alt="Screenshot 2023-03-23 at 12 10 11 AM" src="https://user-images.githubusercontent.com/50941074/227066732-fe1ddd4f-c094-40b8-8a99-1a65e7441cee.png">